python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -859,43 +860,58 @@ def fitDipole(self, source, tol=1e-7, rel_weight=0.1,
             fp = source.getFootprint()
             self.displayFitResults(fp, fitResult)
 
-        fitParams = fitResult.best_values
-        if fitParams['flux'] <= 1.:   # usually around 0.1 -- the minimum flux allowed -- i.e. bad fit.
+        # usually around 0.1 -- the minimum flux allowed -- i.e. bad fit.
+        if fitResult.params['flux'].value <= 1.:
             return None, fitResult
 
-        centroid = ((fitParams['xcenPos'] + fitParams['xcenNeg']) / 2.,
-                    (fitParams['ycenPos'] + fitParams['ycenNeg']) / 2.)
-        dx, dy = fitParams['xcenPos'] - fitParams['xcenNeg'], fitParams['ycenPos'] - fitParams['ycenNeg']
+        # TODO: We could include covariances, which could be derived from
+        # `fitResult.params[name].correl`, but those are correlations.
```

Comment:
`fitResult` should have a `covar` parameter according to [this](https://lmfit.github.io/lmfit-py/fitting.html#minimizerresult-the-optimization-result) that would provide the covariance matrix--no need to use `correl`.